### ISSUE TITLE
Create onboarding docs

### ISSUE THREAD

**Issue by sjn** [CPAN-Security/security.metacpan.org#43]
We need an on-boarding checklist

* Make a reading list, and point people towards it
* Set expectations up front ("I tell you what it's going to be like, so you know what to expect")
* Make tickets suitable for new members, and tag them appropriately
* Allow the new people share what they can and are willing to do, and write this down so others may know who to reach out to when something needs to be discussed
* Tell them about rules we operate under

**Comment by sjn** [comment]
What steps are needed for new members to feel that they are "on their way in" this group? I'm thinking:

* A welcoming message with appropriate tone and language, introducing them to the group purpose, tasks, governing structures and how these may be influenced by them.
* A short reading list of topics that are deemed necessary for them to contribute constructively, and how to help improve this list.
* And introduction to the different forums and information spaces we operate, their purpose, and how to interact with them
    * IRC
    * [Github discussions](https://github.com/orgs/CPAN-Security/discussions) forum
    * Mailing list
    * Mastodon account
    * Website
* A list of easy tasks (tagged so) for them to pick from to get a "quick win" for helping, and how to help keep the task lists up-to-date and useful.
* Have a mentor assigned to them, that they may contact directly for questions and support
* Sign the pre-release disclosure document
* Add them to the relevant organizational access lists, after certain criteria are fulfilled
    * the mailing list
    * the github organization
    * the VINCE group
* A check-list of major efforts they are open to participate in, e.g.
    [ ] Attending PTS, CPANSec gatherings or other sponsored events.
    [ ] Offer their time (20-60% of work week) for paid work on larger CPANSec tasks (assuming available funding)
        * …and eligibility requirements for these
    [ ] Offer to do mentoring

**Comment by stigtsp** [comment]
Just a note about signing the pre-release disclosure agreement: We have to do some vetting/voting as well before allowing someone to access this information. I don't think this should be in the default invite flow.

**Comment by sjn** [comment]
Good point; we probably need two levels of on-boarding – for example _project membership_ and _incident response team membership_?

**Comment by Tux** [comment]
As the Governance project was closed, should this issue be closed or assigned to a new project?

**Comment by sjn** [comment]
I think the closing of the governance project my have been a mistake. I revived it a while ago, do this ticket is still on topic
